### PR TITLE
Added additional knife

### PR DESCRIPTION
**Pull request by GeoMod**
Becuase two knives are better than one.

### CODE REVIEW

File: knife.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-So what the pen is mightier than I. I'll fucking open you up. Make you bleed.
```
Comment (idrez) [inline knife.txt:1]: Why did you remove this line then added it back in.

**Review by idrez — APPROVED** [review]

File: knife.txt (hunk 1 above)
Comment (GeoMod) [reply knife.txt:1]: I don't answer to you! Errr..I sort of do.
